automation/project/Dependencies.scala

```diff
@@ -9,10 +9,13 @@ object Dependencies {
   val workbenchModel: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-model" % workbenchModelV
   val excludeWorkbenchModel = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-model_2.11")
 
-  val workbenchGoogleV = "0.10-d97f551"
+  val workbenchGoogleV = "0.16-eac241e"
```

Comment:
I was getting complaints about `GoogleCredentialMode.Pem` and this made it go away.